docs/source/wavefunction_simulator.rst

```diff
@@ -58,96 +58,64 @@ amplitudes directly or look at a dictionary of associated outcome probabilities.
-    prog.inst(H(0))
-    prog.measure(0, answer_register)
-    print(prog)
-
-Finally we play the game several times
-
-.. code:: python
-
-    wf_sim.run_and_measure(prog, [0, 1], 10)
-
-Remember that the first number is the outcome of the game (value of the answer_register) whereas the second number is the
-outcome of Picard’s decision (value of the picard_register).
-
-No matter what Picard does, Q will always win!
-
-
+Multi-Qubit Basis Enumeration
+-----------------------------
+
+The Rigetti QVM enumerates bitstrings such that qubit `0` is the least significant bit (LSB)
+and therefore on the right end of a bitstring as shown in the table below which contains some
+examples.
+
+=============== ============= ===== ========= ========= =========
+ bitstring       qubit_(n-1)   ...   qubit_2   qubit_1   qubit_0
+=============== ============= ===== ========= ========= =========
+  1...101                  1   ...         1         0         1
+  0...110                  0   ...         1         1         0
+=============== ============= ===== ========= ========= =========
+
+
+
+This convention is counter to that often found in the quantum computing literature where
+bitstrings are often ordered such that the lowest-index qubit is on the left.
+The vector representation of a wavefunction assumes the "canonical" ordering of basis elements.
+I.e., for two qubits this order is ``00, 01, 10, 11``.
+In the typical Dirac notation for quantum states, the tensor product of two different degrees of
+freedom is not always explicitly understood as having a fixed order of those degrees of freedom.
+This is in contrast to the kronecker product between matrices which uses the same mathematical
+symbol and is clearly not commutative.
+This, however, becomes important when writing things down as coefficient vectors or matrices:
+
+.. math::
+
+    \ket{0}_0 \otimes \ket{1}_1 = \ket{1}_1 \otimes \ket{0}_0
+    = \ket{10}_{1,0} \equiv \begin{pmatrix} 0 \\ 0 \\ 1 \\ 0 \end{pmatrix}
+
+As a consequence there arise some subtle but important differences in the ordering of wavefunction
+and multi-qubit gate matrix coefficients.
+According to our conventions the matrix
+
+.. math::
+
+    U_{\rm CNOT(1,0)} \equiv
+    \begin{pmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & 1 & 0 & 0 \\
+        0 & 0 & 0 & 1 \\
+        0 & 0 & 1 & 0
+    \end{pmatrix}
+
+corresponds to the Quil instruction ``CNOT(1, 0)`` which is counter to how most other people in the
+field order their tensor product factors (or more specifically their kronecker products).
+In this convention ``CNOT(0, 1)`` is given by
+
+.. math::
+
+    U_{\rm CNOT(0,1)} \equiv
+    \begin{pmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & 0 & 0 & 1 \\
+        0 & 0 & 1 & 0 \\
+        0 & 1 & 0 & 0
+    \end{pmatrix}
+
+For additional information why we decided on this basis ordering check out our note
+:ref:`*Someone shouts, "|01000>!" Who is Excited?* <https://arxiv.org/abs/1711.02086>`_.
```

Comment:
This is the third line I made a change to, where this had previously read:
*heres our note* [2]

[2] http://